util/cinc-install-1.1.0.sh

```diff
@@ -355,11 +375,12 @@ tmp_dir="$tmp/install.sh.$$"
 channel="stable"
 project="cinc"
 
-while getopts pnv:c:f:P:d:s:l:a opt
+while getopts pnv:b:c:f:P:d:s:l:a opt
 do
   case "$opt" in
 
     v)  version="$OPTARG";;
+    b)  bucket="$OPTARG";;
```

Comment:
Can we add a short inline comment to explain this option a bit more? Something like, "Download CINC client from specified s3 bucket"

Reply:
Yes! Done